plugins/azure-devops/src/routes.ts

```diff
@@ -16,6 +16,11 @@
 
 import { createRouteRef } from '@backstage/core-plugin-api';
 
+export const azurePullRequestDashboardRouteRef = createRouteRef({
+  id: 'azure-pull-request-dashboard',
+  path: '',
```

Comment:
No need to pass the `path` here, it's deprecated

Reply:
Removed path.